docs/manual/en/introduction/Libraries-and-Plugins.html

```diff
@@ -106,6 +106,7 @@ <h3>Wrappers and Frameworks</h3>
 			<li>[link:https://aframe.io/ A-Frame]</li>
 			<li>[link:https://lume.io/ Lume] - HTML elements for 3D graphics built on Three.</li>
 			<li>[link:https://github.com/pmndrs/react-three-fiber react-three-fiber] - React components for 3D graphics built on Three.</li>
+			<li>[link:https://threepipe.org/ threepipe] - A versatile 3D viewer framework using three.js for rendering.</li>
```

Comment:
Can you also add the new link to the translated pages?

Reply:
Sure

Reply:
Never mind. I'll merge and clean up 👍 .

Reply:
Thanks :)